src/Linters/GroupUseStatementsLinter.hack

```diff
@@ -270,6 +270,10 @@ final class GroupUseStatementsLinter extends AutoFixingASTLinter {
 
         if (
           Dict\filter($result[$kind][$namespace][1], ($n) ==> {
+            invariant(
+              $n is KeyedContainer<_, _>,
```

Comment:
what is it if it's a string and what is it if it's a vec? Does sense to refactor to make the type of the variable consistent, instead of using `is`/`as`?